src/main/java/mate.academy/Main.java

```diff
@@ -10,6 +11,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
+        @Inject
         ProductService productService = null;
```

Comment:
You should not assign `null` to `productService` when using dependency injection. The `Injector` should take care of creating and providing an instance of `ProductService`. Remove the `null` assignment.